events_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"github.com/veandco/go-sdl2/sdl"
+	"testing"
+	"time"
+)
+
+// TestCloseEventStream tests if closing our event stream works
+func TestCloseEventStream(t *testing.T) {
+	es := &SdlEventStream{}
+	events := es.Receive()
+	t.Log("Opened channel, calling close in background")
+	errc := make(chan error)
+	// do it in background to give it time
+	go func() {
+		errc <- es.Close()
+	}()
+	select {
+	case err := <-errc:
+		if err != nil {
+			t.Errorf("An error occured: %v", err)
+		}
+		// we have finished in time
+	case <-time.After(3 * time.Second):
```

Comment:
shorten timeouts. Local machine timeouts should be in the one or two digit millisecond range. Extend if necessary. This makes your tests much faster.